File: CS/HideableGroupRowFooters/CustomGridControl/MyGridView.cs
using DevExpress.XtraGrid;
using DevExpress.XtraGrid.Views.Grid;

namespace HideableGroupRowFooters
{
    public class MyGridView : GridView
    {
        public MyGridView(GridControl ownerGrid)
            : base(ownerGrid)
        {

        }

        public MyGridView()
            : base()
        {

        }

        public delegate void CustomFooterVisibleHandler(object sender, HideFooterEventArgs e);
        public event CustomFooterVisibleHandler CustomFooterVisible;
        public virtual void OnCustomFooterVisible(HideFooterEventArgs e)
        {
            if (CustomFooterVisible != null)
            {
                CustomFooterVisible(this, e);
            }

        }



        protected override string ViewName
        {
            get
            {
                return "MyGridView";
            }
        }

    }
}

File: VB/HideableGroupRowFooters/CustomGridControl/MyGridView.vb
Imports DevExpress.XtraGrid
Imports DevExpress.XtraGrid.Views.Grid

Namespace HideableGroupRowFooters
	Public Class MyGridView
		Inherits GridView

		Public Sub New(ByVal ownerGrid As GridControl)
			MyBase.New(ownerGrid)

		End Sub

		Public Sub New()
			MyBase.New()

		End Sub

		Public Delegate Sub CustomFooterVisibleHandler(ByVal sender As Object, ByVal e As HideFooterEventArgs)
		Public Event CustomFooterVisible As CustomFooterVisibleHandler
		Public Overridable Sub OnCustomFooterVisible(ByVal e As HideFooterEventArgs)
			RaiseEvent CustomFooterVisible(Me, e)

		End Sub



		Protected Overrides ReadOnly Property ViewName() As String
			Get
				Return "MyGridView"
			End Get
		End Property

	End Class
End Namespace